lib/mtif/posts.rb

```diff
@@ -125,20 +125,26 @@ def mtif_value(value)
       value.kind_of?(Time) ? value.strftime(DATE_FORMAT) : value
     end
 
-    def convert_to_native_type(raw_value)
+    def convert_to_native_type(raw_value, key)
+      if MULTILINE_KEYS.include?(key)
```

Comment:
I think passing the key is essential to the predictability of the program. Without the key, we can never be 100% sure that the value needs conversion. For example, if I type `"01/31/2002 03:31:05 PM"` in the title, converting it would be wrong.

I don't get quite understand your planned refactoring, but as long as it depends on the key I feel it will be good.